playbooks/provision_vm/provisioning_vm_from_scratch.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
I wouldn't call the playbook as `provisioning_vm_from_scratch.yml` I would call it just `manage_vm` or something similar. We will have specific use cases based on user variables.